wandb/proto/wandb_api.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package wandb_internal;
+
+import "wandb/proto/wandb_base.proto";
+
+option go_package = "core/pkg/service_go_proto";
+
+message ApiRequest {
+  oneof request {
+    ReadRunHistoryRequest read_run_history = 1;
+  }
+}
+
+message ApiResponse {
+  oneof response {
+    ReadRunHistoryResponse read_run_history = 1;
+  }
+}
+
+// Start of Read run history
+
+message ReadRunHistoryRequest {
```

Comment:
```suggestion
message ReadRunHistoryApiRequest {
```

not to confuse with the regular requests since it's still a part of the wandb_internal package.